PatternGenerator.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include "LeakedObjectDetector.h"
+#include "HarmonicSet.h"
+#include "Distortion.h"
```

Comment:
Don't include what you don't use.  There is no `Distortion` or `HarmonicSet` used in this class, so don't include them.
- [ ] revise